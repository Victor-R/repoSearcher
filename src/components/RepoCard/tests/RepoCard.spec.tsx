import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import { RepoCard } from "..";

describe("Components: RepoCard", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <RepoCard title="TestTitle" description="lorem ipsum" link="test" />,
      { wrapper: BrowserRouter }
    );

    expect(container).toMatchSnapshot();
  });
});
