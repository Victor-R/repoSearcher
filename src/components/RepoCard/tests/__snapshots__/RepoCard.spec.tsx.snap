// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components: RepoCard should match snapshot 1`] = `
.c0 {
  background: white;
  padding: 25px 35px;
  transition: all 0.3s ease;
  text-decoration: none;
  border-radius: 10px;
  background: #2b2a33;
  color: white;
  display: inline-flex;
  align-items: center;
}

.c0 svg {
  width: 50px;
  height: 50px;
  fill: #612ea0;
}

.c0>div {
  margin-left: 25px;
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.c0:hover {
  filter: brightness(1.5);
  cursor: pointer;
}

.c1 {
  font-size: 2em;
}

.c2 {
  font-size: 0.765em;
}

<div>
  <a
    class="c0"
    href="/test"
  >
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M3 2.75A2.75 2.75 0 0 1 5.75 0h14.5a.75.75 0 0 1 .75.75v20.5a.75.75 0 0 1-.75.75h-6a.75.75 0 0 1 0-1.5h5.25v-4H6A1.5 1.5 0 0 0 4.5 18v.75c0 .716.43 1.334 1.05 1.605a.75.75 0 0 1-.6 1.374A3.251 3.251 0 0 1 3 18.75ZM19.5 1.5H5.75c-.69 0-1.25.56-1.25 1.25v12.651A2.989 2.989 0 0 1 6 15h13.5Z"
      />
      <path
        d="M7 18.25a.25.25 0 0 1 .25-.25h5a.25.25 0 0 1 .25.25v5.01a.25.25 0 0 1-.397.201l-2.206-1.604a.25.25 0 0 0-.294 0L7.397 23.46a.25.25 0 0 1-.397-.2v-5.01Z"
      />
    </svg>
    <div>
      <h3
        class="c1"
      >
        TestTitle
      </h3>
      <p
        class="c2"
      >
        lorem ipsum
      </p>
    </div>
  </a>
</div>
`;
